test/utilities/miscellaneous.ts

```diff
@@ -20,18 +20,24 @@ export async function pause(duration: Duration = Duration.seconds(1)): Promise<v
 }
 
 export function log(message: string): void {
-  if (EnvironmentSettings.getInstance().logLevel !== 'silent') {
+  if (EnvironmentSettings.getInstance().logLevel === 'warn') {
```

Comment:
Given ['trace', 'debug', 'info', 'warn', 'error', 'silent']
the logLevel can be one of ['warn', 'error']